lambdas/appointment_email.py

```diff
@@ -30,6 +31,12 @@ def lambda_handler(event, context):
     else:
         requester = None
     
+    if len(comme) != 0 :
```

Comment:
Why are you using two variables, comme and comment which are essentially the same with just a prefix in comme at start? Couldn't you just do 
`comment = "Comment: " + (event['comment'].strip() or "No Comment")`

You could remove the strip() in case that isn't how this function is supposed to work. 

In places where you want to use comment directly, just use event['comment'].
Since you aren't doing any logic on comment variable to make it differ from event['comment'], it doesn't make any sense to use another variable just to point to the variable to which we already have access to, specially when there is only once place where it is used, which is just right under the declaration.

Reply:
Done!